test/hotspot/jtreg/runtime/NMT/CommitOverlappingRegions.java

```diff
@@ -49,10 +48,9 @@ public static void main(String args[]) throws Exception {
 
         long addr = wb.NMTReserveMemory(8*size);
 
-        String pid = Long.toString(ProcessTools.getProcessId());
         ProcessBuilder pb = new ProcessBuilder();
 
-        pb.command(new String[] { JDKToolFinder.getJDKTool("jcmd"), pid, "VM.native_memory", "detail"});
+        pb.command(new PidJcmdExecutor().getCommandLine("VM.native_memory", "detail"));
         System.out.println("Address is " + Long.toHexString(addr));
```

Comment:
Q: Can we use the `NMTTestUtils.startJcmdVMNativeMemory()` here?

Reply:
The test uses created ProcessBuilder many times, so the test would require much more changes

Reply:
Okay, thanks.